frontend/src/Store/Actions/seasonPassActions.js

```diff
@@ -42,8 +62,28 @@ export const defaultState = {
     },
     {
       name: 'seriesType',
-      label: 'Series Type',
-      type: filterBuilderTypes.EXACT
+      label: 'Type',
```

Comment:
Let's move all the common filter builder props to `seriesActions` and remove the dupes from the 3 other action files.

Reply:
Done